selfservice/hook/session_issuer.go

```diff
@@ -53,11 +52,6 @@ func (e *SessionIssuer) ExecutePostRegistrationPostPersistHook(w http.ResponseWr
 }
 
 func (e *SessionIssuer) executePostRegistrationPostPersistHook(w http.ResponseWriter, r *http.Request, a *registration.Flow, s *session.Session) error {
-	s.AuthenticatedAt = time.Now().UTC()
```

Comment:
@hperl did you check that the session can not be modified by any hooks, in which case we would need to update it here?

Reply:
Yes we checked, the hooks all modify the identity, the session is not modified by any hook.